bot/filters.py

```diff
@@ -36,10 +36,13 @@ def __init__(self, bot: Bot):
         self.bot = bot
 
     async def __call__(self, callback: CallbackQuery):
-        if callback.message == None:
-            return False
-        chat_state = await load_state(chat_id=callback.message.chat.id)
-        return {"chat_state": chat_state}
+        if type(callback.message) == Message:
```

Comment:
I think it's cleaner to do like this:

```
match message := callback.message:
  case Message(): ...
```

Reply:
`callback.message` may be of type InacessbleMessage which seems to be inherited from Message but doesn't have message_thread_id property. 
`case Message():` accepts inherited classes, so this may cause an error